### PR TITLE
add parsertype to schema, add open source custom attributes to README

### PR DESCRIPTION
**Pull request by mfaquan**
This is for the Fortify 22.2 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +43,21 @@
     * If project.version ends with `-SNAPSHOT`, the artifacts will end up at https://s01.oss.sonatype.org/content/repositories/snapshots/
     * Release versions are published to an OSSRH staging repository and checked for Maven Central requirements
         * Log in to https://s01.oss.sonatype.org/ with the same account used during publishing to verify the artifacts and release them to Maven Central
+
+## Open Source Issues in Fortify Software Security Center
+
+* For SSC to consider issues to be "Open Source", the parser plugin must add the element "&lt;parser-type&gt;DEPENDENCY_SCAN&lt;/parser-type&gt;" to the &lt;issue-parser&gt; element in the plugin's plugin.xml file.
```
Comment (thurmon) [inline README.md]:
```suggestion
* For SSC to consider issues as "Open Source" issues, the parser plugin must add the element "&lt;parser-type&gt;DEPENDENCY_SCAN&lt;/parser-type&gt;" to the &lt;issue-parser&gt; element in the plugin's plugin.xml file.
```

File: README.md [hunk 2]
```diff
@@ -39,3 +43,21 @@
...
+
+* The parser plugin must also include these custom attributes for the vulnerabilities:
+	- externalId: An ID for the issue, like a CVE ID
```
Comment (thurmon) [inline README.md]:
```suggestion
	- externalId: An ID, such as a CVE ID, for the issue
```

File: README.md [hunk 3]
```diff
@@ -39,3 +43,21 @@
...
+	- cwes: Comma-separated list of CWE ID numbers(Only the numbers)
+	- externalUrl: An external URL for the vulnerability
```
Comment (thurmon) [inline README.md]:
```suggestion
	- externalUrl: External URL for the vulnerability
```

File: README.md [hunk 4]
```diff
@@ -39,3 +43,21 @@
...
     * Release versions are published to an OSSRH staging repository and checked for Maven Central requirements
         * Log in to https://s01.oss.sonatype.org/ with the same account used during publishing to verify the artifacts and release them to Maven Central
+
+## Open Source Issues in Fortify Software Security Center
+
+* For SSC to consider issues to be "Open Source", the parser plugin must add the element "&lt;parser-type&gt;DEPENDENCY_SCAN&lt;/parser-type&gt;" to the &lt;issue-parser&gt; element in the plugin's plugin.xml file.
+
+* The parser plugin must also include these custom attributes for the vulnerabilities:
+	- externalId: An ID for the issue, like a CVE ID
+	- cwes: Comma-separated list of CWE ID numbers(Only the numbers)
```
Comment (thurmon) [inline README.md]:
```suggestion
	- cwes: Comma-separated list of CWE IDs (numbers only)
```

File: README.md [hunk 5]
```diff
@@ -39,3 +43,21 @@
...
+	- externalUrl: An external URL for the vulnerability
+	- componentPurl: Package URL for the vulnerable component in the form "pkg:type/namespace/name@version"
+	- componentNamespace: Namespace of the vulnerable component.  This should match the namespace in the package URL
+	- componentName: Name of the vulnerable component.  This should match the name in the packge URL
+	- componentVersion: Version of the vulnerable component.  This should match the name in the package URL
+	- componentPackageType: Package type of the vulnerable component.  This should match the type in the package URL
+	- componentLicenses: The type of licenses for the vulnerable component
```
Comment (thurmon) [inline README.md]:
```suggestion
	- componentLicenses: Type of licenses for the vulnerable component
```

File: README.md [hunk 6]
```diff
@@ -39,3 +43,21 @@
...
+	- invoked:  true/false value for whether or not the analyzed code calls this specific vulnerability
```
Comment (thurmon) [inline README.md]:
```suggestion
	- invoked:  true/false value for whether or not the analyzed code calls this specific issue
```

File: README.md [hunk 7]
```diff
@@ -39,3 +43,21 @@
...
+	- controllable: true/false value for whether or not the analyzed code calls this specific vulnerability with user-controlled data
```
Comment (thurmon) [inline README.md]:
```suggestion
	- controllable: true/false value for whether or not the analyzed code calls this specific issue with user-controlled data
```

File: README.md [hunk 8]
```diff
@@ -39,3 +43,21 @@
...
+	- evidence: Any supporting evidence that the analyzed code is vulnerable to this specific vulnerability
```
Comment (thurmon) [inline README.md]:
```suggestion
	- evidence: Any supporting evidence that the analyzed code is vulnerable to this specific issue
```
OR
"Any supporting evidence that the analyzed code is susceptible to this specific issue"

**Review by thurmon — CHANGES_REQUESTED** [review]
A few edits suggested.

File: README.md (hunk 8 above)
Comment (mfaquan) [reply README.md]: Updated.

**Comment by arungitan** [comment]
one of the commits says 1.1.  be sure to either squash or update commit to say 1.2.

**Comment by arungitan** [comment]
also need to update the README.md to make it consistent with versioning and other details.

**Comment by mfaquan** [comment]
> one of the commits says 1.1. be sure to either squash or update commit to say 1.2.

Version is 1.1.0.  I originally put 1.0.2, but, we decided to bump the minor version and make it 1.1.0.

**Comment by psmf** [comment]
Superseded by #11

**Pull request closed without merging**